AUTHORS

```diff
@@ -1,3 +1,3 @@
 Nicholas H.Tollervey (ntoll@ntoll.org)
 Matt Wheeler (m@funkyhat.org)
-
+Edmond Locard (L0C4RD@gmail.com)
```

Comment:
Hi @L0C4RD and many thanks for your patience.

I completely understand and totally sympathise with your reasons for a low profile. I hope you, in turn, can see why I may also have been a tad paranoid about accepting contributions for code relating to a project aimed at kids from an anonymous third party. 

Given the links you provided above I'm completely happy to accept your patch and any further contributions you may make!

I *really* enjoyed your presentation. It had me grinning like a Cheshire Cat. I have a few questions for you about the talk and I wonder if you might contact me via ntoll <at> ntoll <dot> org.

Also, you're right about our intentions regarding contributions: radical inclusion is one way to say we want to work in a community that reflects all of us. Diversity is a strength!

Finally, there's a few things I want to fix in the code you've submitted. Did you `make check`..? ;-) There's a bunch of code style warnings that I've fixed locally on my machine. Also, there are some Python 2/3 compatibility issues (`StringIO` etc...).

Also, you've introduced a dependency on the third party `IntelHex` module. Does this need to be used? One of the "features" of uflash is that it's stand-alone so you could just drop the file into any other project (like Mu) and "it should just work" (tm). 

Thoughts..?